### PR TITLE
Add 'event' binding to componentCollection in applybindings.ts

### LINKED ISSUE #80: Add `event` binding to `refComponents`
The `event` binding already implemented in `refComponent` (and the `ComponentParams` type), but not in `refComponents`.

It should be added to the `componentCollection` section in `applyBindings.ts`.

### PR DESCRIPTION
**Pull request by jspolancor**
- Add 'event' binding to componentCollection in applybindings.ts
- Add tests for the feature

Fixes #80

### CODE REVIEW

File: src/lib/bindings/applyBindings.test.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    element.append(button);
+    document.body.appendChild(element);
+
+    const handleClick = jest.fn();
+
+    defineComponent({
+      name: 'my-component',
+      refs: {
+        button: refElement('button'),
```
Comment (ThaNarie) [inline src/lib/bindings/applyBindings.test.ts:20]: The test is saying `should apply event bindings to a component`. but it's testing a `refElement`. You should instead be testing `refComponent`.

File: src/lib/bindings/applyBindings.test.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    element.setAttribute('data-component', 'my-component');
+    firstButton.setAttribute('data-component', 'button-component');
+    secondButton.setAttribute('data-component', 'button-component');
+    thirdButton.setAttribute('data-component', 'button-component');
+
+    element.append(firstButton);
+    element.append(secondButton);
+    element.append(thirdButton);
+    document.body.appendChild(element);
```
Comment (ThaNarie) [inline src/lib/bindings/applyBindings.test.ts:55]: Wouldn't it be easier to just set the `innerHTML` to a html string instead?

**Review by ThaNarie — CHANGES_REQUESTED** [review]
Implementation itself looks good 👍

File: src/lib/bindings/applyBindings.test.ts (hunk 1 above)
Comment (jspolancor) [reply src/lib/bindings/applyBindings.test.ts]: 😄 Fixed!

**Review by ThaNarie — APPROVED** [review]

**Pull request merged**